Polyhedron/demo/Polyhedron/CMakeLists.txt

```diff
@@ -125,10 +125,10 @@ if(CGAL_ACTIVATE_CONCURRENT_MESH_3 OR ENV{CGAL_ACTIVATE_CONCURRENT_MESH_3})
 else(CGAL_ACTIVATE_CONCURRENT_MESH_3 OR ENV{CGAL_ACTIVATE_CONCURRENT_MESH_3})
   option(LINK_WITH_TBB
          "Link with TBB anyway so we can use TBB timers for profiling" ON)
-  if(LINK_WITH_TBB)
+  if(LINK_WITH_TBB AND NOT TBB_FOUND)
     find_package(TBB)
     include(CGAL_TBB_support)
-  endif(LINK_WITH_TBB)
+  endif(LINK_WITH_TBB AND NOT TBB_FOUND)
```

Comment:
Why is it necessary?

Reply:
because otherwise there is a risk (it's the case in my configuration) to call find_package(TBB) twice, leading to the following error :
```
CMake Error at C:/dev/tbb2019_20181203oss_win/tbb2019_20181203oss/cmake/TBBConfig.cmake:83 (add_library):
  add_library cannot create imported target "TBB::tbb" because another target
  with the same name already exists.
```

I am making a PR specifically for that, based on 5.1.x-branch

Reply:
Oops. That is a bug in earlier versions of TBB. It was [fixed in TBB 2019 update 5](https://github.com/oneapi-src/oneTBB/commit/112ade3da541b73b053635b6f29cafe4abe4fa5e#diff-44200c17e3e2482035a9fb3e08be1224edf61baedfa4ceb1cda73818911cd1ffR65). That is a bad interaction with https://github.com/CGAL/cgal/pull/5687, that made the module `FindTBB.cmake` from CGAL search for TBB in `Config` mode, first.

@maxGimeno How about adding a version argument to our `find_package NO_MODULE`? That would be:
```cmake
find_package(TBB 2019.5 QUIET NO_MODULE)
```

Reply:
That way, our `FindTBB.cmake` would search for TBB >= 2019 update 5 in Config mode, and otherwise using the older version.